app/views/admin/documentations/_form.html.haml

```diff
@@ -0,0 +1,11 @@
+.field
+  %h4= f.label 'Nom'
+  = f.text_field :name, class: 'form-control', id: 'Nom', required: true, autofocus: true
+.field
+  %h4= f.label 'Abbréviation'
+  = f.text_field :short_name, class: 'form-control', id: 'Abbréviation', required: true
+.field
+  %h4= f.label 'Accès'
```

Comment:
My bad: I realise `cover_path` might be misunderstood.
This attributes holds the URL to the book's cover image. This image would be used as a thumbnail on the mobile app side. :)